ImageKitSample/Program.cs

```diff
@@ -0,0 +1,426 @@
+
+            #endregion
+
+            #region FileVersionRequest
+            DeleteFileVersionRequest delRequest = new DeleteFileVersionRequest
+            {
+                FileId = "file_Id",
+                VersionId = "version_Id",
+            };
+            ResultNoContent resultNoContent1 = imagekit.DeleteFileVersion(delRequest);
+
+            // RestoreFileVersion
+            Result result = imagekit.RestoreFileVersion("file_Id", "version_Id");
+
+            // GetFileVersions
+            ResultFileVersions resultFileVersions = imagekit.GetFileVersions("file_Id");
+
+            // GetFileVersionDetails
+            ResultFileVersionDetails resultFileVersionDetails = imagekit.GetFileVersionDetails("file_Id", "version_Id");
+
+            #endregion
+
+            #region ManageFolder
+
+            CreateFolderRequest createFolderRequest = new CreateFolderRequest
+            {
+                FolderName = "folder_name",
+                ParentFolderPath = "source/folder/path",
+            };
+            ResultEmptyBlock resultEmptyBlock = imagekit.CreateFolder(createFolderRequest);
+
+            // DeleteFolderRequest
+            DeleteFolderRequest deleteFolderRequest = new DeleteFolderRequest
+            {
+                FolderPath = "source/folder/path/folder_name",
+            };
+            ResultNoContent resultNoContent2 = imagekit.DeleteFolder(deleteFolderRequest);
+
+            // CopyFolder
+            CopyFolderRequest cpyFolderRequest = new CopyFolderRequest
+            {
+                SourceFolderPath = "path_1",
+                DestinationPath = "path_2",
+                IncludeFileVersions = true,
+            };
+
+            ResultOfFolderActions resultOfFolderActions = imagekit.CopyFolder(cpyFolderRequest);
+
+            // MoveFolder
+            MoveFolderRequest moveFolderRequest = new MoveFolderRequest
+            {
+                SourceFolderPath = "path_1",
+                DestinationPath = "path_2",
+            };
+
+            ResultOfFolderActions resultOfFolderActions1 = imagekit.MoveFolder(moveFolderRequest);
+
+            #endregion
+
+            #region GetBulkJobStatus
+
+            ResultBulkJobStatus resultBulkJobStatus = imagekit.GetBulkJobStatus("job_Id");
+
+            #endregion
+
+            #region  Purge
+
+            ResultCache resultCache = imagekit.PurgeCache("url");
+
+            ResultCacheStatus resultCacheStatus = imagekit.PurgeStatus("request_Id");
+
+            #endregion
+
+            #region  Metadata
+            ResultMetaData resultMetaData = imagekit.GetFileMetadata("file_Id");
+
+            ResultMetaData resultMetaData1 = imagekit.GetRemoteFileMetadata("https://ik.imagekit.io/demo/medium_cafe_B1iTdD0C.jpg");
+
+            // CustomMetaDataFields
+            ResultCustomMetaDataFieldList resultCustomMetaDataFieldList = imagekit.GetCustomMetaDataFields(true);
```

Comment:
The parameter name `includeDeleted` is not mentioned anywhere in the readme also this should follow same convention as other methods above.